### PR TITLE
Use elevated actor to merge forward-mergers

### PR DESCRIPTION
**Pull request by AyodeAwe**
This solves the problem of `rapids-bot`'s failing attempts to merge forward-merger PRs. See https://nvidia.slack.com/archives/CJWBS14NB/p1710956582188569?thread_ts=1710857430.929599&cid=CJWBS14NB for context behind this implementation.

Test: 
PR failing due to checks/pending review: https://github.com/rapidsai/literate-octo-potato/pull/817
PR merged with elevated actor: https://github.com/rapidsai/literate-octo-potato/pull/822

### CODE REVIEW

File: .github/workflows/deploy.yaml [hunk 1]
```diff
@@ -47,3 +47,4 @@ jobs:
           APP_ID: ${{ secrets.APP_ID }}
           WEBHOOK_SECRET: ${{ secrets.WEBHOOK_SECRET }}
           PRIVATE_KEY: ${{ secrets.PRIVATE_KEY }}
+          GPUTESTER_PAT: ${{ secrets.GPUTESTER_PAT }}
```
Comment (ajschmidt8) [inline .github/workflows/deploy.yaml:50]: is this secret configured correctly for this repository already?

**Review by ajschmidt8 — APPROVED** [review]

File: .github/workflows/deploy.yaml (hunk 1 above)
Comment (AyodeAwe) [reply .github/workflows/deploy.yaml:50]: I intend to do that once we approve the PR

Comment (raydouglass) [reply .github/workflows/deploy.yaml:50]: I've added it

**Review by raydouglass — APPROVED** [review]

**Comment by AyodeAwe** [comment]
I will now disable forward mergers on Jenkins and then deploy this PR

**Pull request merged**